include/mbedtls/ssl_cache.h

```diff
@@ -123,6 +123,23 @@ int mbedtls_ssl_cache_set(void *data,
                           size_t session_id_len,
                           const mbedtls_ssl_session *session);
 
+/**
+ * \brief          Remove the cache entry by the session ID
+ *                 (Thread-safe if MBEDTLS_THREADING_C is enabled)
+ *
+ * \param data            The SSL cache context to use.
+ * \param session_id      The pointer to the buffer holding the session ID
+ *                        associated to \p session.
+ * \param session_id_len  The length of \p session_id in bytes.
+ *
+ * \return                0: The cache entry for session with provided ID
+ *                           is removed or does not exist.
+ *                        1: Internal error.
```

Comment:
I did this because some public functions in `ssl_cache.c` return `1` to indicate an internal error in the module. I'm fine with returning an `MBEDTLS_ERR_xxx` error code.

Reply:
Oh, I see. So that's a preexisting bug. Fixing it is out of scope, but let's at least not expand it.